examples/fakegroundstation/test_ground_station_service.py

```diff
@@ -0,0 +1,70 @@
+import time
+
+import grpc_testing
+import unittest
+
+from google.protobuf.timestamp_pb2 import Timestamp
+from stellarstation.api.v1.groundstation import groundstation_pb2
+from stellarstation.api.v1 import transport_pb2
+
+from fakegroundstation.ground_station_service import GroundStationServiceServicer
+
+
+class TestGroundStationServicer(unittest.TestCase):
+    def setUp(self) -> None:
+        servicers = {
+            groundstation_pb2.DESCRIPTOR.services_by_name['GroundStationService']: GroundStationServiceServicer()
+        }
+
+        self.test_server = grpc_testing.server_from_dictionary(
+            servicers, grpc_testing.strict_real_time()
+        )
+
+    def test_request(self) -> None:
+        request = groundstation_pb2.ListPlansRequest(
+            ground_station_id="2",
+            aos_after=Timestamp(seconds=int(time.time()) - 120, nanos=0),
+            aos_before=Timestamp(seconds=int(time.time() + 3600), nanos=0),
```

Comment:
Can you make it a little more clear to readers what `int(time.time()) - 120` and `int(time.time()) + 3600` mean?